src/components/indicators/indicators.scss

```diff
@@ -38,6 +38,16 @@
     color: #888;
 }
 
+.listItemMediaInfo + .timerIndicator {
```

Comment:
This is probably better suited to be in `src/components/listview/listview.scss`.

Reply:
Moved this to suggested scss.